web/assets/js/app/Browser/component/controller/NodeController.js

```diff
@@ -3,16 +3,18 @@ define([
 ], function(angular) {
     'use strict';
 
-    function NodeController($scope, $state, $graph, $search, $fuzzyFilter, $notification, $treeFactory, $timeout, $q) {
+    function NodeController($scope, $state, $graph, $search, $fuzzyFilter, $treeFactory, $timeout, $q, $success, $error, $errorFromResponse) {
```

Comment:
"in the future" sounds bad in my ears. If you don't need it now, don't develop it now™.

For now you're just adding a little more complexity, with no added value. It's dead code. I suggest you separate the two services when you actually have a feature using it.